src/idpi/tasking.py

```diff
@@ -0,0 +1,18 @@
+"""functionality for tasking and parallel computing."""
```

Comment:
Perhaps the implementation of this module should be moved to PR #86

Reply:
yes, that was a mistake. It belongs to #86